utils/s2n_blob.h

```diff
@@ -17,14 +17,16 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include <stdbool.h>
 
 struct s2n_blob {
     uint8_t *data;
     uint32_t size;
     uint32_t allocated;
-    uint8_t mlocked;
+    uint8_t mlocked :1, growable :1;
```

Comment:
nit:

two lines is clearer, and always include type as above:

```
    uint8_t mlocked :1;
    uint8_t growable :1;
```

Reply:
Done